MoreLinq/TrySingle.cs

```diff
@@ -0,0 +1,79 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2019 James Webster. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+namespace MoreLinq
+{
+    using System;
+    using System.Collections.Generic;
+
+    partial class MoreEnumerable
+    {
+        /// <summary>
+        /// Determines the cardinality of the source sequence, provides a
```

Comment:
This summary needs re-phrasing to closely align with `TrySingle` as the name of the method. It should not start by saying that it determines the cardinality of the sequence but rather that it _attempts_ to retrieve and project a result from a sequence of one element; and failing that, it provides the cardinality of the sequence for the zero or many cases.

Reply:
I've updated, but had to put a fairly ugly and long `<see cref=""/>` to `Enumerable.Single()` to explain the method.